sdk/textanalytics/Azure.AI.TextAnalytics/src/ClientCommon.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.AI.TextAnalytics.Models;
+using Azure.Core.Pipeline;
+
+namespace Azure.AI.TextAnalytics
+{
+    internal static class ClientCommon
+    {
+        public static async ValueTask<RequestFailedException> CreateExceptionForFailedOperationAsync(bool async, ClientDiagnostics diagnostics, Response response, IReadOnlyList<TextAnalyticsErrorInternal> errors, string errorMessage = default)
```

Comment:
could u add a test for this? I want to make sure we are surfacing the right information here

Reply:
not sure I missed it, but can't find the test